src/main/resources/default/assets/common/core.js.pasta

```diff
@@ -167,6 +167,23 @@ sirius.postJSON = function (url, params) {
     });
 }
 
+/**@
+ * Extracts the values of all form elements inside a form.
+ * <p>
+ * This attempts to replace jQuery's serialize() / serializeArray() method.
+ *
+ * @param _node the DOM-node of the <form> to get the values from
+ * @returns a map of name-value pairs, mapping the name of the input to its value
+ */
+sirius.getFormData = function (_node) {
+    const formData = new FormData(_node);
+    const object = {};
+    for (const key of formData.keys()) {
```

Comment:
`const` in for-loop declaration is not supported in IE11: https://caniuse.com/const